### ISSUE TITLE
Hi, I have some questions about the classifier training

### ISSUE THREAD

**Issue by indussky8** [edgarschnfld/CADA-VAE-PyTorch#4]
Hi, at line 411 of vaemodel.py, train_X includes 'train_unseen_feat'. I think 'train_unseen_feat' is from the 'test_unseen' data by analyzing data_loader.py, and is it right? If so, why can we use the test unseen data and its labels in the classifier training? 
Looking forward to your kind reply.
Best,

**Comment by edgarschnfld** [comment]
Hi, this is for few-shot learning. In the case of _n_ shots, _n_ examples per class are moved from the test to the training set. Those are the 'train_unseen_feat' from the 'test_unseen' data. :)
